tests/config/pytest_modules.yml

```diff
@@ -2235,6 +2235,10 @@ ucsc/bedgraphtobigwig:
   - modules/ucsc/bedgraphtobigwig/**
   - tests/modules/ucsc/bedgraphtobigwig/**
 
+ucsc/bedtobigbed:
+  - modules/ucsc/bedtobigbed/**
+  - tests/modules/ucsc/bedtobigbed/**
+
```

Comment:
Sorry, this one is my fault. I renamed this module recently, and didn't think of changing its position in tests/config/pytest_modules.yml
Thanks !